packages/wdio-crossbrowsertesting-service/tests/launcher.test.js

```diff
@@ -0,0 +1 @@
+
```

Comment:
You have to provide unit test coverage otherwise this service can't be merged

Reply:
I'm currently working to add unit test.